org.eclipse.xtext.common.types.eclipse.tests/tests/org/eclipse/xtext/common/types/access/xtext/ui/JdtBasedConstructorScopeTest.java

```diff
@@ -50,6 +54,7 @@ public void setUp() throws Exception {
 	}
 	
 	@Test public void testGetContents_01() {
+		assumeTrue(JavaCore.getPlugin().getBundle().getVersion().compareTo(new Version(3,31,0)) >= 0);
```

Comment:
Interesting. This does no longer work with older versions of JDT?

Reply:
it looked like it and i dont understand why

Reply:
=> needs to be looked at

Reply:
https://github.com/eclipse/xtext/issues/2056#issuecomment-1532439336